pygsp/graphs/nngraphs/bunny.py

```diff
@@ -34,7 +34,5 @@ def __init__(self, **kwargs):
             'distance': 8,
         }
 
-        super(Bunny, self).__init__(Xin=data['bunny'],
-                                    epsilon=0.02, NNtype='radius',
-                                    center=False, rescale=False,
+        super(Bunny, self).__init__(data['bunny'], kind='radius', radius=0.02,
```

Comment:
Do we want a radius graph by default for the bunny?

Reply:
That comes from the matlab version. It was however updated to a kNN in version 0.7.0 https://github.com/epfl-lts2/gspbox/commit/60790478280e989010d4c9dd4acb45d6bf52043c. Do you think we should do it here too? If yes, a small PR with justification is best.